### ISSUE TITLE
Obsolete enforcer

### ISSUE THREAD

**Issue by dcostea** [ignatandrei/RSCG_TimeBombComment#3]
An interesting use case can be an obsolete "enforcer."
Let's assume we have a piece of code (a class, struct, method, property) which we plan to refactor later. Most of the time we forget about that, but no more! Now we can put an inoffensive comment in the code which turns into an exception at compile-time after the date and time mentioned in the comment.
The exception message can be a generic message or a custom message.

E..g.
```
//OBSOLETE_ENFORCER Warning 2021/09/15 11:11:11 We had enough of this! let's refactor it soon!
//OBSOLETE_ENFORCER Error 2021/09/15 11:11:11 We had enough of this! let's refactor it now!
```

**Comment by ignatandrei** [comment]
Could be in collision with
[Obsolete("asda",true)
https://docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=net-5.0

Let me think how we can combine those ( obsolete is sealed)

**Comment by ignatandrei** [comment]
see next comment

~~//OBSOLETE_ENFORCER 2021-09-15~~ ( see next comment)
~~[Obsolete("this will be obsolete soon",Err_FullName)]~~( see next comment)
public string FullName()


and generate the

const bool Err_FullName= false;// or true after dates

( the Warning or Error is not necessary, default will be false = warning, after date will be error )

( see next comment)

**Comment by ignatandrei** [comment]
The final form will be in the form TB_yyyyMMdd 

[Obsolete("this will be obsolete soon",TB_20210914)] 
public string FullName()

and generate the

const bool TB_FullName= false;// or true after dates

**Comment by ignatandrei** [comment]
solved in nuget version 2021.9.15.2200

https://www.nuget.org/packages/RSCG_TimeBombComment/2021.9.15.2200
